Jamulus.pro

```diff
@@ -745,6 +745,7 @@ DISTFILES += ChangeLog \
     src/res/fronticon.png \
     src/res/fronticonserver.png \
     src/res/mixerboardbackground.png \
+    src/res/1x1.png \
```

Comment:
It's not just _any_ 1x1 pixel, is it.  Can it have a more meaningful name, please?

Reply:
Yeh, valid point. But since it could be used in multiple ways, I'll just call it `transparent1x1.png`.